packages/neuron-ui/src/components/Send/hooks.ts

```diff
@@ -108,17 +110,32 @@ const updateTransactionWith =
             }
             return res.result
           }
-          if (res.status === 0 || res.status === 114) {
-            throw new Error(res.message.content)
+
+          if (res.status === 0) {
+            setErrorMessage(res.message.content)
+            throw new Error()
+          }
+          if (res.status === 114) {
+            const num = res.message.content.match(/\d+/g)?.[0]
+            const errors = [] as string[]
+            items.forEach(({ amount }) => {
+              if (!amount || amount < num) {
+                errors.push(res.message.content)
+              } else {
+                errors.push('')
+              }
+            })
+            setItemsErrorMessage(errors)
+            throw new Error()
```

Comment:
I guess it will be better to verify it before `generate tx`. 
Additionally, it may have a problem, one address's least ckb is 61, and another address's least ckb is 63.